app/service/rest.py

```diff
@@ -1322,3 +1328,38 @@ def get_service_join_request(request_id: uuid.UUID):
         raise InvalidRequest(message=f"Service join request with ID {request_id} not found.", status_code=404)
 
     return jsonify(service_join_request.serialize()), 200
+
+
+@service_blueprint.route("/update-service-join-request-status/<uuid:request_id>", methods=["PUT"])
```

Comment:
Any reason of using `PUT`? :thinking: We have been using mainly `POST` in notify and this API is required to update partial data

Reply:
changed to POST